Migrations.cs

```diff
@@ -81,6 +81,16 @@ public int UpdateFrom1()
                         Hint = "Fix background in place to give a faux parallax effect.",
                         Label = "Fix Background Pattern"
                     })
+                )
+                .WithField("BackgroundFill", field => field

```

Comment:
This should probably be in a separate method `UpdateFrom`, otherwise any sites that have ran `UpdateFrom1` will not get this change.

Reply:
Ah okay... In practice, would that look like this... ?

```
_contentDefinitionManager.AlterPartDefinition("AnimationPart", part => part
                .Attachable()
                .WithDescription("Define settings for animating widget.")
                .WithDisplayName("Animation")
                .WithField(

Reply:
@Newnab Something like this.

```
public int UpdateFrom5()
{
    _contentDefinitionManager.AlterPartDefinition("BackgroundPart", part => part
        .WithField("BackgroundFill", field => field
            .OfType(nameof(BooleanField))
            .WithDisplayName("Background Fill")
            .WithPosition("4")
            .WithSettings(new BooleanFieldSettings
            {
                Hint = "By default, background styles will tile and repeat. In some cases, you may want to force the background to fill the available space and not repeat. To do so, tick this box. Please note that depending on the chosen background and content, this option may result in backgrounds being 'stretched'.",
                Label = "Background Fill"
            })
        ));

    return 6;
}
```